roles/openstack_cinder/tasks/main.yml

```diff
@@ -1,5 +1,5 @@
 ---
-
+##############test##############################################
```

Comment:
What sort of test are you looking to do here?

Reply:
@jtaleric I wanted to check if the job is getting triggered if there is any change in the roles/openstack* .It is just for testing